plugins/ruby-foreman-dynflow/debian/changelog

```diff
@@ -1,3 +1,7 @@
+ruby-foreman-dynflow (0.7.8-1) stable; urgency=low
+
+  * Update to 0.7.8
+
 ruby-foreman-dynflow (0.7.7-1) stable; urgency=low
```

Comment:
Missing a line with your name + date here.  `scripts/changelog.rb` may help you.